eZ/Bundle/EzPublishCoreBundle/Resources/config/slot.yml

```diff
@@ -19,116 +19,131 @@ parameters:
     ezpublish.http_cache.signalslot.trash.class: eZ\Publish\Core\MVC\Symfony\Cache\Http\SignalSlot\TrashSlot
     ezpublish.http_cache.signalslot.recover.class: eZ\Publish\Core\MVC\Symfony\Cache\Http\SignalSlot\RecoverSlot
 
+    ezpublish.http_cache.signalslot.publish_content_type.class: eZ\Publish\Core\MVC\Symfony\Cache\Http\SignalSlot\PublishContentTypeSlot
+    ezpublish.http_cache.signalslot.delete_content_type.class: eZ\Publish\Core\MVC\Symfony\Cache\Http\SignalSlot\DeleteContentTypeSlot
+
 services:
-    ezpublish.http_cache.signalslot.http_cache:
+    # Content
+    ezpublish.http_cache.signalslot.abstract_content:
         abstract: true
-        class: "%ezpublish.http_cache.signalslot.http_cache.class%"
-        arguments: ["@ezpublish.http_cache.purger"]
+        class: "%ezpublish.http_cache.signalslot.abstract_content.class%"
```

Comment:
If service declaration is abstract, you don't need to set a class as it will never be instanciate
(working ex: https://github.com/prestaconcept/PrestaCMSCoreBundle/blob/master/Resources/config/block.xml#L18)